dozer-k8/dozer-api-svc.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
File names seem to repeat a lot. Can we remove the prefix ?

Reply:
Done